.circleci/config_template.yml

```diff
@@ -257,6 +257,10 @@ j8_separate_jobs: &j8_separate_jobs
         requires:
           - start_j8_unit_tests
           - j8_build
+    - j8_unit_tests-oa:
```

Comment:
All the JUnit-based test jobs have a `_repeat` companion job, such as `j8_unit_tests_repeat`, `j8_utests_cdc_repeat`, etc. We need to add analogous `j8_utests_oa_repeat` and `j11_utests_oa_repeat` jobs. 

The script `generate.sh` should also be updated to consider those new jobs.

Please let me know if you need any help with this.

Reply:
First I want to get your +1 on the approach of the feature flag and overall approach. Then we ping Benedict.

If we're +1 then a rebase is needed which will bring jdk17 and other circle changes where I will add all those oa changes, yes, but after the rebase. They are extremely painful and I don't want to do them unless I have a +1 on the rest from you both.

So wdyt, besides the circle loose ends, are you +1 here?

Reply:
You have my +1 on the approach :)

Reply:
I was going to look into this and then decided everything is too nosy and cluttered already. We should use `repeat_ant` for all these variations imo.

At the beginning that was nice sugar-coating but now it's growing and it will only get bigger imo. Circle config files are painful enough. I would leave it as it is and open a ticket to drop all ant based repeats, use the single ant repeat always and document it properly :shrug: That should pay itself over time with the shortened config files. Wdyt?

Reply:
The changes to CircleCI on [`CASSANDRA-14227-trunk-deleteme`](https://github.com/bereng/cassandra/commits/CASSANDRA-14227-trunk-deleteme) look good to me. We definitively need the flag to skip the detection of tests, but that's something that we should probably do in a separate ticket since it's needed in all branches.

As for testing this patch for flakies, there are 129 modified tests so it will probably be too much for a single push. We can take the same approach that we used for SAI, which was in a similar situation. It consists on using `generate.sh` to produce the list of modified tests, split it into smaller batches and produce some CircleCI pushes by manually passing the test batches in `REPEATED_UTESTS`. I'd say 4-5 batches will do. I can do that once it's rebased and ready for the final round of tests.

Reply:
I'll create a ticket for the -s.

And yes, after the rebase, I will manually choose all tests that have changes that are more than a simple int/long toggle. Then I'll batch them up for repeated testing.